security/firely-auth/firely-auth-tutorial.rst

```diff
@@ -145,7 +123,24 @@ Note that we also set the Audience in the Advanced Settings to the default value
 
 .. image:: /images/auth_postman_encode_secret.png
 
-Step 5 - Connect Firely Server to Firely Auth
+Step 6 - Users
+^^^^^^^^^^^^^^
+
+With the required settings in place, you can start Firely Auth in PowerShell by running::
+
+    > ./Firely.Auth.Core.exe
+
+And you can access it with a browser on ``https://localhost:5001``. It will use a self-signed certificate by default, for which your browser will warn you.
```

Comment:
I would suggest to link to mkcert and recommend creating a local cert.

Reply:
for a production environment they should not use mkcert, but i will add it with a note it being for testing

Reply:
added a kestrel setup section